src/main/mbar.c

```diff
@@ -375,9 +524,19 @@ void ExamDispInit(void) {
     vsAnimationInit();
 }
 
-INCLUDE_ASM("asm/nonmatchings/main/mbar", ExamDispPlySet);
+void ExamDispPlySet(GLOBAL_PLY *ply, int pos) {
+    exam_global_ply[pos] = ply;
+}
 
-INCLUDE_ASM("asm/nonmatchings/main/mbar", ExamDispReq);
+void ExamDispReq(int ply, int plmi) {
+    exam_disp_cursor_timer = 0;
+    exam_global_ply_current = exam_global_ply[ply];
+    if (plmi != 0) {
```

Comment:
`plmi` is a flag (known by context) so you can remove the explicit `!= 0` check (`if (plmi)`).